packages/badge/src/spectrum-badge.css

```diff
@@ -0,0 +1,398 @@
+        var(--spectrum-global-color-static-magenta-600)
+    );
+    --spectrum-badge-m-textonly-text-color-hover: var(
+        --spectrum-badge-m-magenta-textonly-text-color-hover,
+        var(--spectrum-global-color-static-white)
+    );
+    --spectrum-badge-m-textonly-background-color-hover: var(
+        --spectrum-badge-m-magenta-textonly-background-color-hover,
+        var(--spectrum-global-color-static-magenta-700)
+    );
+}
+:host([variant='purple']) {
+    --spectrum-badge-m-texticon-text-color: var(
+        --spectrum-badge-m-purple-texticon-text-color,
+        var(--spectrum-global-color-static-white)
+    ); /* .spectrum-Badge--purple */
+    --spectrum-badge-m-textonly-background-color: var(
+        --spectrum-badge-m-purple-textonly-background-color,
+        var(--spectrum-global-color-static-purple-600)
+    );
+    --spectrum-badge-m-textonly-text-color-hover: var(
+        --spectrum-badge-m-purple-textonly-text-color-hover,
+        var(--spectrum-global-color-static-white)
+    );
+    --spectrum-badge-m-textonly-background-color-hover: var(
+        --spectrum-badge-m-purple-textonly-background-color-hover,
+        var(--spectrum-global-color-static-purple-700)
+    );
+}
+:host([variant='seafoam']) {
+    --spectrum-badge-m-texticon-text-color: var(
+        --spectrum-badge-m-seafoam-texticon-text-color,
+        var(--spectrum-global-color-static-white)
+    ); /* .spectrum-Badge--seafoam */
+    --spectrum-badge-m-textonly-background-color: var(
+        --spectrum-badge-m-seafoam-textonly-background-color,
+        var(--spectrum-global-color-static-seafoam-600)
+    );
+    --spectrum-badge-m-textonly-text-color-hover: var(
+        --spectrum-badge-m-seafoam-textonly-text-color-hover,
+        var(--spectrum-global-color-static-white)
+    );
+    --spectrum-badge-m-textonly-background-color-hover: var(
+        --spectrum-badge-m-seafoam-textonly-background-color-hover,
+        var(--spectrum-global-color-static-seafoam-700)
+    );
+}
+:host([variant='yellow']) {
+    --spectrum-badge-m-texticon-text-color: var(
+        --spectrum-badge-m-yellow-texticon-text-color,
+        var(--spectrum-global-color-static-black)
+    ); /* .spectrum-Badge--yellow */
+    --spectrum-badge-m-textonly-background-color: var(
+        --spectrum-badge-m-yellow-textonly-background-color,
+        var(--spectrum-alias-background-color-yellow-default)
+    );
+    --spectrum-badge-m-textonly-text-color-hover: var(
+        --spectrum-badge-m-yellow-textonly-text-color-hover,
+        var(--spectrum-global-color-static-black)
+    );
+    --spectrum-badge-m-textonly-background-color-hover: var(
+        --spectrum-badge-m-yellow-textonly-background-color-hover,
+        var(--spectrum-alias-background-color-yellow-hover)
+    );
+}
+:host {
+    background-color: var(--spectrum-badge-m-textonly-background-color);
+    color: var(--spectrum-badge-m-texticon-text-color); /* .spectrum-Badge */
+}
+:host(:hover) {
```

Comment:
Did they say whether they would be removing hover with the move to a non-interactive model? If so, do we want to disabled that now so it doesn't become "expected" at the consumer level?

Reply:
Moved to `spectrum-config.js`:

```
/* disable :hover until Spectrum CSS disables hover upstream */
            exclude: [/\:hover/],
```